src/parameters/parameters_flow_solver.h

```diff
@@ -30,6 +31,10 @@ class FlowSolverParam
 
     bool steady_state; ///<Flag for solving steady state solution
 
+    /** Name of the output file for writing the sensitivity data;
+        will be written to file: sensitivity_table_filename.txt */
+    std::string sensitivity_table_filename;
```

Comment:
It's only used in conjunction with the FlowSolver, so probably yeah. It's pretty much analogous to `unsteady_data_table_filename` but contains info for my tests. It's only present in Flow Solver cases, so if your flow solver case doesn't use it then it can just be ignored.